src/Stripe.net/Entities/Accounts/Account.cs

```diff
@@ -89,52 +56,29 @@ internal object InternalBusinessLogo
         [JsonProperty("external_accounts")]
         public StripeList<IExternalAccount> ExternalAccounts { get; set; }
 
-        [JsonProperty("keys")]
-        public CustomAccountKeys Keys { get; set; }
-
-        [JsonProperty("legal_entity")]
-        public LegalEntity LegalEntity { get; set; }
+        [JsonProperty("individual")]
+        public Person Individual { get; set; }
 
         [JsonProperty("metadata")]
         public Dictionary<string, string> Metadata { get; set; }
 
-        [JsonProperty("payout_schedule")]
-        public PayoutSchedule PayoutSchedule { get; set; }
-
-        [JsonProperty("payout_statement_descriptor")]
-        public string PayoutStatementDescriptor { get; set; }
-
         [JsonProperty("payouts_enabled")]
         public bool PayoutsEnabled { get; set; }
 
-        [JsonProperty("product_description")]
-        public string ProductDescription { get; set; }
+        // TODO: Figure out if we can use Requirements as there's `current_deadline`
```

Comment:
I think we need to rename `Requirements` to `PersonRequirements` and have a separate `AccountRequirements`.

Reply:
Yep, I was just hoping not to have to do that. It's annoying to have so many of those resources with similar names and one property that changes.